### PR TITLE
Feature/hmw 13 show restoration protection plans on map

### PR DESCRIPTION
**Pull request by cschwinderg**
## Related Issues:
* https://jira.epa.gov/browse/HMW-13
* https://jira.epa.gov/browse/HMW-59
* https://jira.epa.gov/browse/HMW-5 - addresses comment about hiding surrounding waterbodies layer when WSIO layer is on

## Main Changes:
* Added GIS data to the map for the `Restoration Plans` tab of the `Restore` panel.
* Added GIS data to the map for the `Protection Projects` accordion item of the `Protect` tab.
* Added attains protection projects data to the right hand list for the `Protection Projects` accordion item of the `Protect` tab.
* Fixed an issue where users could highlight waterbodies on the surrounding waterbodies layer by hovering over them. This caused weird issues where waterbodies would be highlighted even though the waterbodies layer was turned off.
* Added code to hide the surrounding waterbodies layer when the WSIO layer is turned on.

## Steps To Test:
1. Navigate to http://localhost:3000/community/040302020902/restore
2. Click on the `Restoration Plans` tab
3. Verify there are no items in the list with a `Plan Type` of `Protection Approach`.
4. Verify that waterbodies are shown on the map in blue
5. Click on a waterbody and verify a popup loads. The popup will be a list of projects/plans associated with that waterbody
6. Click on the `Protect` panel
7. Expand the `Protection Projects` accordion
8. Verify that all items in the list have a `Plan Type` of `Protection Approach`.
9. Verify that waterbodies are shown on the map in blue
10. Click on a waterbody and verify a popup loads. The popup will be a list of projects/plans associated with that waterbody

## TODO:
- [ ] Need to get EPA input on what to name the toggles (Attains Plans and GRTS Projects) in the `Protection Projects` accordion item of the `Protect` tab, or whether or not these toggles are even needed.
- [ ] Get EPA guidance on what they would like to see in the popups. The `Plan Type` item isn't really necessary on the `Protect` panel.

### CODE REVIEW

File: app/client/src/components/pages/Community/components/tabs/Protect.js [hunk 1]
```diff
@@ -181,40 +208,110 @@ function Protect() {
...
+        total_319_funds: project.total_319_funds,
+        project_start_date: project.project_start_date,
+        status: project.status,
+        project_link: project.project_link,
+        watershed_plans: project.watershed_plans,
+        completionDate: '',
+        documents: [],
+        organizationIdentifier: '',
+        organizationName: '',
+        organizationTypeText: '',
```
Comment (courtneymyers) [inline app/client/src/components/pages/Community/components/tabs/Protect.js:240]: @cschwinderg  we should be consistent in using camelCase vs underscores in the property names – I'd suggest camelCase, as that's what we've used for all variable names across the project.

```suggestion
        source: 'grts',
        title: project.prj_title,
        id: project.prj_seq,
        pollutants: project.pollutants,
        total319Tunds: project.total_319_funds,
        projectStartDate: project.project_start_date,
        status: project.status,
        projectLink: project.project_link,
        watershedPlans: project.watershed_plans,
        completionDate: '',
        documents: [],
        organizationIdentifier: '',
        organizationName: '',
        organizationTypeText: '',
```

Comment (courtneymyers) [reply app/client/src/components/pages/Community/components/tabs/Protect.js:240]: @cschwinderg no action on your part right now – I'll push up an update for this.

Comment (cschwinderg) [reply app/client/src/components/pages/Community/components/tabs/Protect.js]: Makes sense. I committed the same change around the same time as you. It doesn't look like that merge broke anything.

Comment (courtneymyers) [reply app/client/src/components/pages/Community/components/tabs/Protect.js]: Haha, I saw that – no worries... that's partially my fault – I meant to start a review and accidentally clicked leave a single comment. It's just double fixed now :)

File: app/client/src/components/shared/MessageBoxes/index.js [hunk 2]
```diff
@@ -100,9 +88,7 @@ const noteBoxStyles = css`
 `;
 
 export {
-  StyledTextBox, // TODO: remove
   StyledInfoBox, // TODO: remove
-  StyledSuccessBox, // TODO: remove
```
Comment (courtneymyers) [inline app/client/src/components/shared/MessageBoxes/index.js:105]: Great!

**Review by courtneymyers — APPROVED** [review]
Looks great, Caleb! I pushed up some small changes (mostly formatting/renaming of a few variables). Can you test on your end? If all is still working as expected, please feel free to merge this in. Thanks!

**Pull request merged**